README.md

```diff
@@ -94,6 +95,60 @@ prime.
 3. As an initiator or participant, you now click on proceed. If running in automated mode, EasySMPC will automatically perform all steps until the final result is displayed. If running in manual mode, all users need to send and receive e-mails prepared by EasySMPC to perform the computation.
 4. The final perspective shows the result of the secure addition of all variables.
 
+## Command line version
```

Comment:
Hier ist noch ein bisschen sprachliches Polishing notwendig. Beispiele (habe nur Stichproben angesehen):

- "use the jar easy-smpc-cli-*{Version}*.jar as a creator and as two participants or more" -> Kein sinnvoller Satz.
-  "The parameters mean to the following:" -> "to" muss weg.

Reply:
Ok, wie sollen die Sätze lauten und welche sind noch zu ändern?